src/main/scala/inox/ast/TypeOps.scala

```diff
@@ -306,6 +308,7 @@ trait TypeOps {
       case Untyped => Some(0)
       case BooleanType => Some(2)
       case UnitType => Some(1)
+      // TODO BVType => 2^size
```

Comment:
Ok, for now I leave this as is and we can look into it later if we feel it's relevant.